static/app/main.tsx

```diff
@@ -16,9 +17,11 @@ function Main() {
         <Router
           history={browserHistory}
           render={props => (
-            <RouteContext.Provider value={props}>
-              <RouterContext {...props} />
-            </RouteContext.Provider>
+            <RouteAnalyticsContextProvider {...props}>
```

Comment:
Why do you need this, can't you just consume from the `RouteContext`?

Reply:
@evanpurkhiser Yea maybe that would work, I'd have to investigate that. In either case, not too worried about that part right now.